### PR TITLE
Added compatibility for Wagtail >= 2.7, and updated testsuite to support Python >= 3.6

### PR DESCRIPTION
**Pull request by dirkmoors**
Added some minor fixes to make Wagtailvideos support Wagtail >= v2.7

### CODE REVIEW

**Comment by dirkmoors** [comment]
@seb-b @SableWalnut @timheap Can you review, merge and push a new release?

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,32 @@
...
-      python: 3.5
-    - env: TOXENV=py35-dj21-wt25
-      python: 3.5
-    - env: TOXENV=py36-dj20-wt25
-      python: 3.6
-    - env: TOXENV=py36-dj21-wt25
-      python: 3.6
-    # - env: TOXENV=py36-dj22-wt25
-      # python: 3.6
-    - env: TOXENV='flake8,isort'
```
Comment (seb-b) [inline .travis.yml]: Can you add this line back in for testing flake8 + isort 

Some of your files that have `import wagtail` at the top will fail the isort test

Comment (seb-b) [reply .travis.yml]: wait nevermind I see it's added in tox.ini

**Comment by seb-b** [comment]
Thanks for this, released as part of `2.8.0`

**Pull request merged**